manifests/0000_90_kube-controller-manager-operator_05_alerts.yaml

```diff
@@ -39,3 +39,27 @@ spec:
           for: 15m
           labels:
             severity: critical
+        - alert: GarbageCollectorSyncFailed
+          annotations:
+            summary: There was a problem with syncing the resources for garbage collection.
+            description: Garbage Collector had a problem with syncing and monitoring the available resources. Please see KubeControllerManager logs for more details.
+          expr: |
+            rate(garbagecollector_controller_garbagecollector_resources_sync_error_total{}[1h]) > 0
+          labels:
```

Comment:
I have removed the `for`s as they are not necessarily needed

Reply:
just to make sure this is what you want:
```
rate(garbagecollector_controller_garbagecollector_resources_sync_error_total{}[1h]) > 0
```
means that if any sync error occur, the alert will fire right away
whereas:
```
rate(garbagecollector_controller_garbagecollector_resources_sync_error_total{}[5m]) > 0

for: 1h
```
means that if at least one sync error occurs every 5 minutes during the course of an hour, the alert will fire

The first approach seems very noisy but maybe it is intended.

Reply:
I have put `for` period back to `60m` after a discussion with @dgrisonnet since it is not that important to fire an alert right away since non working gc is not that critical. We can report the alert after getting persistent failures for one hour